swagger.json

```diff
@@ -1425,4 +1488,5 @@
             }
         }
     }
+}
     
```

Comment:
@King-Success . Can you add a new line at the end of the file?.

Reply:
Good catch mate :/

Reply:
I think there's a new line there already :/

Reply:
Turns out I had additional space at the bottom, pushed a fix